### ISSUE TITLE
Can't install Bio::DB::BigFile

### ISSUE THREAD

**Issue by cgy0627** [Ensembl/ensembl-vep#1466]
## Describe the issue
I followed the guideline (https://asia.ensembl.org/info/docs/tools/vep/script/vep_download.html#bigfile) exactly, but still can't install Bio::DB::BigFile. Also, every time I try to run `sudo cpanm Bio::DB::BigFile`, it can't seem to find the location of Kent source tree even though I put it in $KENT_SRC (perhaps this is related to the error I'm getting?).

VEP has been successfully installed and runs perfectly.

## Additional information
Please fill in the following sections to help us find the source of your issue as quickly as possible.

### System
- VEP version: 110
- VEP Cache version: 110
- Perl version: 5.32
- OS: 6.1.38-59.109.amzn2023.x86_64
- tabix installed

### Full VEP command line
```
 sudo cpanm Bio::DB::BigFile
```

### Full error message
> cc1: some warnings being treated as errors
error building lib/Bio/DB/BigFile.o from 'lib/Bio/DB/BigFile.c' at /usr/share/perl5/vendor_perl/ExtUtils/CBuilder/Base.pm line 186.
-> FAIL Installing Bio::DB::BigFile failed. See /root/.cpanm/work/1690354176.276223/build.log for details. Retry with --force to force install it.

**Comment by dglemos** [comment]
Hi @cgy0627, 
The dependencies can be tricky to install on a more modern system.
VEP installation instructions do not recommend using `sudo cpanm`. From the [list of instructions](https://www.ensembl.org/info/docs/tools/vep/script/vep_download.html#bigfile) which one did you run differently? I recommend you re-run the installation this time without using sudo.

As an alternative you can use the VEP Docker image. You can find the instructions here: https://www.ensembl.org/info/docs/tools/vep/script/vep_download.html#docker

**Comment by cgy0627** [comment]
The only difference is that I used a different version of Perl.
`mkdir -p $HOME/cpanm
export PERL5LIB=$PERL5LIB:$HOME/cpanm/lib/perl5
cpanm -l $HOME/cpanm Bio::DB::BigFile`

I didn't do this part because previously using perl 5.26 as suggested didn't work out for me due to clashes with glibc(https://github.com/Ensembl/ensembl-vep/issues/1458).
Using the perl that was originally installed on my linux (Perl 5.32) worked perfectly fine for installing VEP (which I used sudo cpanm for), but now I'm having trouble with Bio::DB::BigFile.

I'll let you know after trying VEP installation without sudo.
